spec/fake_braintree/payment_method_spec.rb

```diff
@@ -34,6 +34,26 @@ def token_for(month, year)
   end
 end
 
+describe 'Braintree::PaymentMethod.delete' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.